packages/agreement-process/src/utilities/errorMappers.ts

```diff
@@ -52,7 +52,7 @@ export const submitAgreementErrorMapper = (
     )
     .with("agreementNotFound", () => 404)
     .with("operationNotAllowed", () => 403)
-    .with("agreementAlreadyExists", () => 409)
+    .with("agreementAlreadyExists", "contractAlreadyExists", () => 409)
```

Comment:
In Scala, the management service sends a 409 for this - however, the process service does not map it to another code and probably results in a 500. We choose to send `409` in this case because it's more meaningful.